src/mesh/mesh.cpp

```diff
@@ -1135,10 +1135,12 @@ void Mesh::DoStaticRefinement(ParameterInput *pin) {
             << "Refinement level must be larger than 0 (root level = 0)" << std::endl;
         PARTHENON_FAIL(msg);
       }
-      if (lrlev > max_level) {
+      int numlevel = pin->GetInteger("parthenon/mesh", "numlevel", 1,
+                                     "maximum level of refinement globally");
```

Comment:
I'm getting a build error on this branch **too many arguments to function call, expected at most 3, have 4**, I assume the default `,1` should not be there (with `Get` as opposed to `GetOrAdd`)?